content/en/docs/concepts/cluster-administration/system-logs.md

```diff
@@ -0,0 +1,113 @@
+
+### JSON log format (ALPHA)
+
+By providing `--logging-format=json` flag to commandline we can change format of logs written by Kubernetes System components.
+JSON format allows for much more efficient log generation and is more concise format.
+
+Example of JSON log format (pretty printed):
+```json
+{
+   "ts": 1580306777.04728,
+   "v": 4,
+   "msg": "Pod status updated",
+   "pod":{
+      "name": "nginx-1",
+      "namespace": "default"
+   },
+   "status": "ready"
+}
+```
+**WARNING** Not all logs are guaranteed to be written in JSON format (e.g. during process start). If you intend to parse logs, please make sure to can handle non JSON logs.
+
+Keys with special meaning:
+* `ts` - timestamp as Unix time (required, float)
+* `v` - verbosity (required, int, default 0)
+* `err` - error string (optional, string)
+* `msg` - message (required, string)
+
+**WARNING** AS JSON format is an ALPHA feature, details like name of fields or JSON serialization are a subject to change.
+
+List of components currently supporting JSON format:
+* {{< glossary_tooltip term_id="kube-controller-manager" text="kube-controller-manager" >}}
+* {{< glossary_tooltip term_id="kube-apiserver" text="kube-apiserver" >}}
+* {{< glossary_tooltip term_id="kube-scheduler" text="kube-scheduler" >}}
+* {{< glossary_tooltip term_id="kubelet" text="kubelet" >}}
+
+**WARNING** JSON format doesn't support most preexisting klog flags. Component will exit if provided with both `--logging-format=json` and any unsupported `klog` flag. Please read [command-line-tools-reference] to find what flags are supported.
+
+### Log verbosity level
+
+By controlling verbosity (using `-v` command line flag) we can filter logs based on how severe they are, controlling resulting log volume.
+The higher verbosity of log the less severe they are, with verbosity 0 being used for logs that should ALWAYS be visible.
+The practical default level is 2.
+
+### Logs location
+
+There are two types of system components: those that run in a container and those
+that do not run in a container. For example:
+
+* The Kubernetes scheduler and kube-proxy run in a container.
+* The kubelet and container runtime, for example Docker, do not run in containers.
+
+On machines with systemd, the kubelet and container runtime write to journald. If
+systemd is not present, they write to `.log` files in the `/var/log` directory.
+System components inside containers always write to the `/var/log` directory,
+bypassing the default logging mechanism.
+Similarly to the container logs, system component logs in the `/var/log`
+directory should be rotated. In Kubernetes clusters brought up by
+the `kube-up.sh` script, those logs are configured to be rotated by
+the `logrotate` tool daily or once the size exceeds 100MB.
```

Comment:
```suggestion
On machines with systemd, the kubelet and container runtime write to journald.
Otherwise, they write to `.log` files in the `/var/log` directory.
System components inside containers always write to `.log` files in the `/var/log` directory,
bypassing the default logging mechanism.
Similar to the container logs, you should rotate system component logs in the `/var/log` directory. 
In Kubernetes clusters created by the `kube-up.sh` script, log rotation is configured by the `logrotate` tool. 
The `logrotate` tool rotates logs daily, or once the log size is greater than 100MB. 
```